frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditLayoutSetForSubform/EditLayoutSet/CreateNewLayoutSet/CreateNewLayoutSet.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { renderWithProviders } from '../../../../../../testing/mocks';
+import { CreateNewLayoutSet } from './CreateNewLayoutSet';
+import type { ComponentType } from 'app-shared/types/ComponentType';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+import { screen, waitFor } from '@testing-library/react';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+import { app, org } from '@studio/testing/testids';
+import { QueryKey } from 'app-shared/types/QueryKey';
+import { layoutSets } from 'app-shared/mocks/mocks';
+import type { LayoutSets } from 'app-shared/types/api/LayoutSetsResponse';
+import userEvent from '@testing-library/user-event';
+import type { FormComponent } from '../../../../../../types/FormComponent';
+import { AppContext } from '../../../../../../AppContext';
+import { appContextMock } from '../../../../../../testing/appContextMock';
+
+const onSubFormCreatedMock = jest.fn();
+
+describe('CreateNewLayoutSet', () => {
+  afterEach(jest.clearAllMocks);
+
+  it('displays the card with label and input field', () => {
+    renderCreateNewLayoutSet();
+    const card = screen.getByLabelText(
+      textMock('ux_editor.component_properties.subform.created_layout_set_name'),
+    );
```

Comment:
Can we use `getByRole` with `input` here?

Reply:
fixed